sources/test.sh

```diff
@@ -1157,6 +1161,7 @@ query_plan_test() {
   fi
 
   echo compile empty query plan code
+  rm ${OUT_DIR}/query_plan.o # Make doesn't do a clean build automatically for some reason...
```

Comment:
This is a super weird thing I haven't figured out what's going on - for some reason the `query_plan_test` binary doesn't get updated even though the `query_plan.[c|h|o]` files have been updated to generate the "empty query plan", instead it still outputs the non-empty query plan.

Reply:
it's probably a missing dependency in the Makefile I'll look into it.